exampleCourse/questions/demo/annotated/engMarkovBrewing/question.html

```diff
@@ -1,54 +1,67 @@
 <pl-question-panel>
-    <p>
-        As a food researcher at large beverage company, you have been tasked with coming up with a secret formula for the next probiotic superfood: {{params.beverage}}!
-         At any point in time during the fermentation process, the {{params.beverage}} solution can have <strong>four ingredients</strong> that react with
-         each other: sugar water, acid, carbon dioxide, and the {{params.beverage}} culture itself.
-    </p>
-
-    <pl-figure file-name="drink.png" width="450"></pl-figure>
-    <br>
-
-    <p> You know how each ingredient of the solution changes in one hour: </p>
-    <ul>
-        <li> {{params.sw_to_acid}}% of the sugar water gets converted to acid, and another {{params.sw_to_culture}}% is converted to the culture. </li>
-        <li> {{params.acid_to_co2}}% of the acid is converted to carbon dioxide, and {{params.acid_to_culture}}% is converted to the culture. </li>
-        <li> {{params.co2_to_acid}}% of the carbon dioxide gets re-absorbed back into acid. </li>
-        <li> The culture does not change into another state. </li>
-    </ul>
-
-    <p> Given the initial solution contains <strong>{{params.initial_sw}}% of sugar water and {{params.initial_culture}}% of culture</strong>,
-      find how many hours it will take for the concentration of sugar water to decrease to <strong>{{params.percent_composition}}%</strong>,
-      and save this as <code class="user-output">hours</code>.
-      Save the composition of ingredients at this time as the 1d numpy array <code class="user-output">composition</code>, assuming the
-      following ingredient's order:
-    </p>
-
-    <p>
+  <p>
```

Comment:
Is there any sort of indentation standard for the examples?

Reply:
I don't think so. This was auto formatted with what is set on save in Vscode.

Reply:
Hmm, I saw that these files are ignored in the `.prettierignore`. In my editor the indentation doesn't change.

```
["INFO" - 11:39:43 AM] Using ignore file (if present) at /Users/retep/repos/PrairieLearn/.prettierignore
["INFO" - 11:39:43 AM] File Info:
{
  "ignored": true,
  "inferredParser": null
}
["INFO" - 11:39:43 AM] File is ignored, skipping.
```

Reply:
I was hoping to potentially save the formatting for #11740 - I'm playing around to see if that is possible.

Reply:
Prettier is set up to format HTML files, but .prettierignore ignores these files. The VSCode plugin doesn't always respect .prettierignore, so changing the file ends up reformatting it occasionally.

https://github.com/PrairieLearn/PrairieLearn/blob/24ae27f94b02b5e843cc0daffb551e8b38212e52/.prettierignore#L58-L60

Reply:
We don't auto-format these because Prettier doesn't play nicely with Mustache syntax and our bespoke `<markdown>` tag.

Reply:
(If we're going to reformat this, I'd prefer to manually re-format everything to match our preferred style. There's a bunch of random indentation/spacing that isn't really conventional. If we don't want to fix this here, I lean slightly towards reverting the formatting changes for now.)

Reply:
Should we revert the formatting with the format-on-save removed from the editor config for html files? I'm ok with reverting the formatting changes to what we had before, but this will be automatically re-formatted once someone saves this file again in vscode.

Reply:
It shouldn't, assuming your editor isn't misconfigured: https://github.com/PrairieLearn/PrairieLearn/blob/577bf017b0de082ed2e29f949f57132afbd21c0a/.prettierignore#L59

Reply:
My editor didn't respect this, which is why this was formatted in the first place. (I think @/jonatanschroeder commented about this?)

Reply:
What's the value of this extension setting look like for you?

<img width="485" alt="Screenshot 2025-04-21 at 12 53 38" src="https://github.com/user-attachments/assets/d44a7dfd-1cc3-49bf-8e2f-dd037e74f5a0" />

Reply:
![image](https://github.com/user-attachments/assets/7fb68c1e-13d8-4edc-889a-c21d7909390e)

Reply:
What is the directory you have open? Is it PL itself or a parent/child of it?

Reply:
PL itself

Reply:
Resolved in meeting discussion, changes reverted here: https://github.com/PrairieLearn/PrairieLearn/pull/11817/commits/d7188f39bf1e117ad1e604d658375a81c6ea9c45

Reply:
For visibility, we saw in the meeting that formatting was still looking weird after that commit, we're gonna wait til it's further improved before merging.

Reply:
Fixed it! 👍🏽